x-pack/plugins/reporting/server/lib/create_queue.ts

```diff
@@ -6,10 +6,10 @@
 
 import { ReportingCore } from '../core';
 import { JobSource, TaskRunResult } from '../types';
-import { createTaggedLogger } from './create_tagged_logger'; // TODO remove createTaggedLogger once esqueue is removed
 import { createWorkerFactory } from './create_worker';
 // @ts-ignore
 import { Esqueue } from './esqueue';
+import { createTaggedLogger } from './esqueue/create_tagged_logger';
```

Comment:
I moved this lib to the ESQueue folder so we don't forget to delete it when we delete ESQueue